primefaces/src/main/java/org/primefaces/component/selectoneradio/SelectOneRadioRenderer.java

```diff
@@ -119,9 +119,16 @@ protected void encodeResponsiveLayout(FacesContext context, SelectOneRadio radio
 
         Converter converter = radio.getConverter();
         String name = radio.getClientId(context);
-        int columns = radio.getColumns();
         String currentValue = ComponentUtils.getValueToRender(context, radio);
 
+        int columns = radio.getColumns();
+        if ("lineDirection".equals(layout)) {
```

Comment:
What happens is the label is long? It will probably start wrapping in the 1/12th column, right?

Reply:
If gives it column `md-1` so column 13 should be wrapped on the next line.

Reply:
I'm not talking about 12+ items, but the length of a single label. So instead of "Option1" use "A label which you might expect in everyday use and might be longer".

Reply:
ahh good point let me make a fix.

Reply:
OK updated. The columns actually default to 12 unless you override it so it will be 12 but if you do `columns=2` with long labels it would make them both obviousle `md-6` making them much bigger.

Reply:
I think we should just drop flexing and using columns in that case.

Reply:
I don't think we can else it won't be responsive anymore?